cloudmarker/stores/emailalert.py

```diff
@@ -0,0 +1,81 @@
+"""email store plugin."""
+
+
+import email
+import logging
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+# Define module-level logger.
+_log = logging.getLogger(__name__)
+
+
+class EmailAlert:
+    """A plugin to send email notification for anomalies found."""
+
+    def __init__(self, host, port, subject, to, sender, body):
+        """Create an instance of :class:`EmailAlert` plugin.
+
+        Arguments:
+            host (string): Server hostname for SMTP.
+            port (int): Server port for STMP.
+            subject (string): Subject line for email.
+            to (list): List of recipients(string).
+            sender (string): From field for email.
+            body (string): Email body.
+
+        """
+        self.host = host
+        self.port = port
+        self.subject = subject
+        self.to = to
+        self.sender = sender
+        self.body = body
+        self.stringbuffer = []
+
+    def write(self, record):
+        """Write JSON records to the file system.
+
+        This method is called once for every ``record`` read from a
+        cloud. In this example implementation of a store, we simply
+        send the ``record`` in JSON format via email to the recipient.
+        The records keeps appending to a stringbuffer which then is
+        written in the email body
+        Arguments:
```

Comment:
Note that if we run `make docs` it would automatically modify `docs/api/cloudmarker.stores.rst` to automatically include the documentation of this plugin in the generated documentation. It's a good idea to add this file and commit it too as part of this pull request.